src/test/regress/expected/prepared_statements_3.out

```diff
@@ -0,0 +1,47 @@
+SET search_path TO "prepared statements";
+-- test function evaluation with parameters in an expression
+PREPARE prepared_function_evaluation_insert(int) AS
+	INSERT INTO prepare_func_table (key, value1) VALUES ($1+1, 0*random());
+-- execute 7 times to trigger prepared statement usage
+EXECUTE prepared_function_evaluation_insert(1);
+EXECUTE prepared_function_evaluation_insert(2);
+EXECUTE prepared_function_evaluation_insert(3);
+EXECUTE prepared_function_evaluation_insert(4);
+EXECUTE prepared_function_evaluation_insert(5);
+EXECUTE prepared_function_evaluation_insert(6);
+EXECUTE prepared_function_evaluation_insert(7);
+SELECT key, value1 FROM prepare_func_table ORDER BY key;
```

Comment:
Minor: if we get the same "key"s in the future, this test might become flaky. Currently that is not the case as all the keys are unique

Reply:
Whoever adds new keys should think that :p  I'll add value_1 to the order by :)